snapcraft/tests/test_plugin_gulp.py

```diff
@@ -0,0 +1,147 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from os import path
+from unittest import mock
+
+import snapcraft
+from snapcraft.plugins import gulp, nodejs
+from snapcraft import tests
+
+
+class GulpPluginTestCase(tests.TestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        self.project_options = snapcraft.ProjectOptions()
+
+        patcher = mock.patch('snapcraft.internal.common.run')
+        self.run_mock = patcher.start()
+        self.addCleanup(patcher.stop)
+
+        patcher = mock.patch('snapcraft.sources.Tar')
+        self.tar_mock = patcher.start()
+        self.addCleanup(patcher.stop)
```

Comment:
I'm not sure what tar has to do with this. That means that either it's unnecessary, or missing a comment.